app/src/main/java/com/example/diceroller/DiceViewAdapter.kt

```diff
@@ -1,49 +1,45 @@
 package com.example.diceroller
 
-import android.content.Context
 import android.view.LayoutInflater
 import android.view.ViewGroup
-import androidx.core.graphics.drawable.toDrawable
 import androidx.recyclerview.widget.RecyclerView
 import com.example.diceroller.databinding.ItemRolledDiceBinding
 import java.text.DateFormat
 
-class DiceViewAdapter (
-    private val items: List<Dice>
-    ): RecyclerView.Adapter<DiceViewAdapter.DiceViewHolder>() {
-
-    class DiceViewHolder(private val itemRolledDiceBinding: ItemRolledDiceBinding) :
```

Comment:
This being here made for some confusing reading. Its fine, my preference is to have nested classes after all the functions of the main class.

Reply:
I can see why, saw some code where they took it out of the class for better readability and then saw that instead the documentation just have it before the functions so I preferred to follow it instead because it seemed more organized when you take the nested class as a property, which may be wrong now that I say it.
The placement comes down to preference right? or is there a best practice for nested classes?